src/components/DonutChart.js

```diff
@@ -242,16 +241,12 @@ const styles = {
     transform: 'translate(-50%, -50%)'
   },
   label: {
-    color: '#333',
-    fontSize: '14px',
-    fontWeight: '800'
-  },
-  pointer: {
-    cursor: 'pointer'
+    color: '#999',
+    fontSize: '0.4em',
```

Comment:
I am setting the base font size in px using the a calculation based on the chart dimensions. The ems are just used to scale down the label text. I can set them all with pixels, but I think it would make it less responsive. Thoughts?